.github/workflows/workflow.yaml

```diff
@@ -3,12 +3,24 @@ name: Build and Release Rust Binaries
 on:
   workflow_call:
     inputs:
-      executable_name:
-        description: 'The name of the executable'
-        required: true
+      binary_name:
```

Comment:
it is a breaking change that will require upstream updates - we only use it in 3 or 4 places currently, so probably better to do it now than later

Reply:
I did see that it seems to be recommended to create a repo literally named `.github` in your org for org-level workflows. We making this one able to be open sourced, but is there any reason for us to consider making a `.github` repo?

Reply:
I like each workflow to be versioned separately